pkg/framework/plugins/defaultevictor/defaultevictor_test.go

```diff
@@ -326,6 +329,8 @@ func TestDefaultEvictorFilter(t *testing.T) {
 	lowPriority := int32(800)
 	highPriority := int32(900)
 
+	minPodAge := metav1.Duration{50 * time.Minute}
```

Comment:
```
pkg/framework/plugins/defaultevictor/defaultevictor_test.go:332:15: k8s.io/apimachinery/pkg/apis/meta/v1.Duration struct literal uses unkeyed fields
```

`metav1.Duration{Duration: 50 * time.Minute}`

Reply:
Done in https://github.com/kubernetes-sigs/descheduler/pull/1475/commits/c7de3c40c19da27114a6bceab147959c1318eda1